StreamChatSwiftUI.xcodeproj/project.pbxproj

```diff
@@ -1417,7 +1416,6 @@
 		8465FBAB2746873A00AF091E = {
 			isa = PBXGroup;
 			children = (
-				84EE53B22BBC41B600FD2A13 /* stream-chat-swift */,
```

Comment:
I was not able to build since my local checkout has a different name. I removed it for now since we should not have local package references committed (@testableapple probably wants to build without local reference as well).

Reply:
yeah, I pushed that by mistake